WebApp/src/main/java/com/isa/webapp/service/UserDetailsServiceImpl.java

```diff
@@ -10,24 +11,24 @@
 
 @Service
 @RequiredArgsConstructor
+@Slf4j
 public class UserDetailsServiceImpl implements UserDetailsService {
 
-    private static final Logger LOGGER = LogManager.getLogger(UserDetailsServiceImpl.class);
     private final UserService userService;
 
     @Override
     public UserDetails loadUserByUsername(String email) {
-        LOGGER.debug(() -> "Attempting to load user by email: " + email);
+
         try {
             UserDetails user = userService.findUserByEmail(email)
                     .orElseThrow(() -> new UsernameNotFoundException("User not found with email: " + email));
-            LOGGER.info(() -> "User loaded successfully with email: " + email);
+            log.info("User loaded successfully with email: {}", email);
             return user;
         } catch (UsernameNotFoundException e) {
-            LOGGER.error(() -> "User not found with email: " + email + ", error: " + e.getMessage());
+            log.error("User not found with email: {}, error: {}", email, e.getMessage());
             throw e;
         } catch (RuntimeException e) {
-            LOGGER.error(() -> "An unexpected error occurred while loading user by email: " + email + ", error: " + e.getMessage());
+            log.error("An unexpected error occurred while loading user by email: {}, error: {}", email, e.getMessage());
             throw new UsernameNotFoundException("An unexpected error occurred", e);
         }
```

Comment:
tę konstrukcję można uprościć i zostawić tylko to co w try, bo rzucasz tam wyjątek, który ma msg, że nie ma usera, po czym łapiesz wyjątek logując że nie ma usera i rzucasz wyjątek dalej